structured_type_test.go

```diff
@@ -569,6 +600,114 @@ func TestArrayOfObjects(t *testing.T) {
 	})
 }
 
+func TestArrayOfArrays(t *testing.T) {
+	warsawTz, err := time.LoadLocation("Europe/Warsaw")
+	assertNilF(t, err)
+	testcases := []struct {
+		name     string
+		query    string
+		actual   any
```

Comment:
this is more of an `actualType`, right?

Reply:
Actually no. This is a placeholder for actual value.